voicemail-api/config.yml

```diff
@@ -0,0 +1,53 @@
+---
+  title:  Download Voicemail Recordings
+  description: Download Voicemail Recordings using the Voicemail API
+  categories:
+  - Recordings
+  steps:
+  - title: Introduction
+    content: |
+      This tutorial provides users a step by step procedure on how to download Voicemail Recordings using Voicemail API for user or queue.    
+  - title: Authentication
+    content: |
+      Authenticate with Genesys Cloud using Implicit Grant. More information about authentication could be found in the [Authorization Documentation](/api/rest/authorization/).

```

Comment:
Nice.  I like links as it drives content.  Not quite sure yet how are markdown->HTML generation works, but you might want to make sure that the link opens in a new tab.  Otherwise, it will take the user away from the tutorial. 

@PrinceNebulon If you can give me some insight on the behavior I would appreciate it.

Reply:
You can't force a link to open in a new window in markdown.

Reply:
> You can't force a link to open in a new window in markdown.

That makes me sad (you cant event escape and put some Javascript on the link) :)

Reply:
Do I need to remove the links instead?

Reply:
No, it's fine. It's up to the user to choose to open in a new window in this case.